torch/_functorch/aot_autograd.py

```diff
@@ -19,6 +19,9 @@
 from torch import Tensor
 from torch._dispatch.python import enable_python_dispatcher
 from torch._dynamo.utils import dynamo_timed
+from torch._logging.loggable_types import AOTForwardGraphLogRec, AOTJointGraphLogRec, AOTBackwardGraphLogRec
+import torch._logging
```

Comment:
Why do you need the two imports here and below?

Reply:
Yeah don't think this is needed